src/magento-payments/onboarding-payments.md

```diff
@@ -1,3 +1,129 @@
+
+1. On the _Admin_ sidebar go to **Sales** > **Magento Payments**.
+1. Click <span class="btn">Configure Magento Services</span>.
+
+   This option will be visible if you have not yet configured Magento Services for your Magento account.
+
+   You will be directed to the configuration area in the Admin, **Stores** > _Settings_ > **Configuration** > **Magento Services**, to configure your Magento Services.
+
+1. Follow the steps in the [Create Saas Environment]({% link system/saas.md%}#createsaasenv) section of the Magento Services topic to create a SaaS environment (configure Magento Services).
+
+## Enable sandbox testing
+
+There are two options for completing sandbox onboarding:
+
+* **Sign up for a free PayPal Developer's account** and _then_ proceed with our sandbox onboarding flow in the Payments Admin. You can create multiple accounts using this method.
+* **Create a sandbox account on-the-fly** during the onboarding process. This method does not require you to log into PayPal prior, but does require a valid email address.
+
+Either option will aid you in completing onboarding.
+
+To complete sandbox onboarding:
+
+1. On the _Admin_ sidebar go to **Sales** > **Magento Payments**.
+1. Click <span class="btn">Sandbox onboarding</span>.
+
+   This option will be visible if you have not yet completed sandbox onboarding for Magento Payments.
+
+   You are directed to the [PayPal Developer Account page](https://developer.paypal.com/developer/accounts/).
+
+1. Click <span class="btn">Log in to Dashboard</span> and log into an existing PayPal Developers account or click <span class="btn">Sign Up</span> to create a new one.
+1. To create a sandbox account:
+   1. Go to _SANDBOX_ > **Accounts**.
+   1. Click <span class="btn">Create account</span>.
+   1. Select **Business** as the Account Type and click <span class="btn">Create</span>.
+   1. In the _Sandbox Accounts_ section, click the three dots in the _Manage accounts_ column for the sandbox account you just created.
+   1. Click **View/edit account**.
+
+      ![PayPal - View/edit sandbox account]({% link magento-payments/assets/onboarding-viewedit-sandbox.png %}){: .zoom}
+      _View/edit sandbox account_
+
+   1. Copy the Email ID and System Generated Password for future use.
+
+   You are redirected back to the Magento Payments home. The <span class="btn">Sandbox onboarding</span> button will no longer be visible and you see "Sandbox payments pending" text.
+
+   Once your PayPal sandbox onboarding is approved, you will see a notification stating that your payment system is currently in sandbox mode and is not processing live payments.
+
+## Set Magento Payments as payment method
+
+After you [configure your Magento Services](#configure-magento-services) and enable either [sandbox testing](#enable-sandbox-testing) or [live payments](#enable-live-payments) (or both!) you must set Magento Payments as your payment method.
```

Comment:
If we are configuring sandbox payments, should we mention enabling live payments?

Reply:
Good call.